src/index.js

```diff
@@ -0,0 +1,109 @@
+const API_KEY = 'http://localhost:5000';
+
+const increaseTemp = () => {
+    const tempElement = document.querySelector('#tempValue');
+    let currentTemp = Number(tempElement.innerHTML);
+    currentTemp++; 
+    tempElement.innerHTML = currentTemp;
+    setTempColor(currentTemp);
+    setLandscape(currentTemp);
+};
+
+const decreaseTemp = () => {
+    const tempElement = document.querySelector('#tempValue');
+    let currentTemp = Number(tempElement.innerHTML);
+    currentTemp--; 
+    tempElement.innerHTML = currentTemp;
+    setTempColor(currentTemp);
+    setLandscape(currentTemp)
+};
```

Comment:
The logic between these 2 methods are pretty similar. Consider combining them into one method and having some logic to determine if you need to increment or decrement the weather.